vms/registration/password_validate.py

```diff
@@ -0,0 +1,18 @@
+def validate_password(password):
+	upper_case=0 #counter for uppercase characters
```

Comment:
please use `true`/`false` for	`upper_case` `lower_case` `number` and `special_character`. Any particular why you are using integers?

Reply:
@m-murad no, there's no particular reason behind that . I've implemented the changes you had suggested. Please have a look.